.github/workflows/ci-pipeline.yml

```
@@ -7,26 +7,26 @@ on:
     branches: [ main ]
 
 jobs:
-  generation-tests:
+  build-and-test:
     runs-on: ubuntu-latest
 
     steps:
     - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)